app/forms/hyrax/forms/resource_form.rb

```diff
@@ -69,6 +69,8 @@ class Permission < Hyrax::ChangeSet
 
       property :visibility # visibility has an accessor on the model
 
+      property :date_modified, readable: false
+      property :date_uploaded, readable: false
```

Comment:
@no-reply Can you explain why these were added?  I didn't see any mention of them in this PR.

Reply:
eghhh... they're mostly unrelated, but were needed to make some related work work out.

okay to merge with them in place?